### PR TITLE
Disable gray log on Mac, because it is invisible in terminal.

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: bin/main.dart [hunk 1]
```diff
@@ -11,7 +11,7 @@ import 'package:pana/pana.dart';
 final _gray = '\u001b[1;30m';
 final _none = '\u001b[0m';
 
-String gray(text) => "$_gray$text$_none";
```
Comment (kevmoo) [inline bin/main.dart]: I'm on mac and think it's fine!

File: bin/main.dart [hunk 2]
```diff
@@ -11,7 +11,7 @@ import 'package:pana/pana.dart';
 final _gray = '\u001b[1;30m';
```
Comment (kevmoo) [inline bin/main.dart]: Change `30m` to `37m` – see how it looks?

Comment (kevmoo) [inline bin/main.dart]: Per http://tldp.org/HOWTO/Bash-Prompt-HOWTO/x329.html

File: bin/main.dart (hunk 1 above)
Comment (isoos) [inline bin/main.dart]: I'm using `Terminal.app` in `Pro` mode with semi-transparent background, in which `30m` is not visible, `37m` is better. I guess you are using it in either totally black or white background?

Comment (kevmoo) [inline bin/main.dart]: Totally black

Comment (kevmoo) [inline bin/main.dart]: Or just add a flag for 'no color' or something?

Comment (isoos) [inline bin/main.dart]: At that point could just use `--json`, but it is harder to read :-)
I'm fine overriding this locally, although a bit inconvenient.

Comment (kevmoo) [inline bin/main.dart]: Just add a flag, right?

Or read an environment var?

I like having another color – it lets me distinguish log output from "other" print/stdout/stderr

**Pull request closed without merging**